README.md

```diff
@@ -213,6 +220,90 @@ oneAgentSDK.addCustomRequestAttribute("salesAmount", 2500);
 
 When no service call is being traced, the custom request attributes are dropped.
 
+<a name="webrequests"/>
+
+## Trace web requests
+
+<a name="inwebrequests"/>
+
+### Trace incoming web requests
+
+You can use the SDK to trace incoming web requests. This might be useful if Dynatrace does not support the respective web server framework or language processing the incoming web requests.
+
+To trace an incoming web request you first need to create a WebServerInfo object. The info object represents the endpoint of your web server (web server name, application name and context root). This object should be reused for all traced web requests within for the same application.
+
+```Java
+WebServerInfo wsInfo = OneAgentSDK.createWebServerInfo("WebShopProduction", "CheckoutService", "/api/service/checkout");
```

Comment:
Is "/api/service/checkout" a good example for context root? It could cause confusion because this looks a bit like an actual URL. At least that's what I thought when I read over it.

Reply:
probably not the best example. will fix it, when working over base repo (https://github.com/Dynatrace/OneAgent-SDK#webrequests)